index.html

```diff
@@ -103,6 +103,7 @@ <h2>Membership</h2>
             
         </section>
 
+        <button>Back to Top</button>
```

Comment:
This should be an anchor like that:
```suggestion
        <a href="#">Back to Top</a>
```